state/aws/dynamodb/metadata.yaml

```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: aws.dynamodb
+version: v1
+status: stable
+title: "AWS DynamoDB"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-dynamodb/
+capabilities:
+  - crud
+  - transactional
+  - etag
+  - ttl
+authenticationProfiles:
```

Comment:
Doesn't this component support authentication without access keys too? Not sure if I remember correctly

Reply:
@ItalyPaleAle  This component should support authentication using AWS Environment variables AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, etc...
How do we express this in this metadata schema?

Reply:
Added a new issue to address this: **[Component Metadata Schema] Extend to Support Shared Auth Profile** https://github.com/dapr/components-contrib/issues/2935